### PR TITLE
AL lessons 3 home work

### PR DESCRIPTION
**Pull request by AlexandrLezh**
Please check my homework.

### CODE REVIEW

File: src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package student_aleksandrs_lezhennikovs.lesson_3.level_7;
+
+class Shop {
+    String shopName;
+    String shopAddress;
+    boolean shopOpenStatus;
```
Comment (AlexJavaGuru) [inline src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java:6]: isOpen

**Review by AlexJavaGuru — CHANGES_REQUESTED** [review]

File: src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_5/Dog.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package student_aleksandrs_lezhennikovs.lesson_3.level_5;
+
+class Dog {
+
+    String nickname;
+    String dogsColor;
```
[the remark this answers is not in the document]
Comment (AlexandrLezh) [reply src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_5/Dog.java]: fixed

File: src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package student_aleksandrs_lezhennikovs.lesson_3.level_7;
+
+class Shop {
+    String shopName;
```
[the remark this answers is not in the document]
Comment (AlexandrLezh) [reply src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java]: fixed

File: src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java (hunk 1 above)
Comment (AlexandrLezh) [reply src/main/java/student_aleksandrs_lezhennikovs/lesson_3/level_7/Shop.java]: fixed

**Review by AlexJavaGuru — APPROVED** [review]

**Pull request merged**